country_codes.js

```diff
@@ -0,0 +1,287 @@
+const EUCountries = {
+  Austria:      "AT",
+  Belgium:      "BE",
+  Bulgaria:     "BG",
+  Croatia:      "HR",
+  Cyprus:       "CY",
+  Czechia:      "CZ",
+  Denmark:      "DK",
+  Estonia:      "EE",
+  Finland:      "FI",
+  France:       "FR",
+  Germany:      "DE",
+  Greece:       "GR",
+  Hungary:      "HU",
+  Ireland:      "IE",
+  Italy:        "IT",
+  Latvia:       "LV",
+  Lithuania:    "LT",
+  Luxembourg:   "LU",
+  Malta:        "MT",
+  Netherlands:  "NL",
+  Poland:       "PL",
+  Portugal:     "PT",
+  Romania:      "RO",
+  Slovakia:     "SK",
+  Slovenia:     "SI",
+  Spain:        "ES",
+  Sweden:       "SE",
+};
+
+const UKCountries = {
+  United_Kingdom:   "GB"
+};
+
+const EEACountries = {
+    Iceland:        "IS",
+    Liechtenstein:  "LI",
+    Norway:         "NO"
+}
+
+// const isoCountries = {
```

Comment:
again some commented code :) If you for sure want to have it in here, put some meaningful comment above the section you're commenting out.

Reply:
This is a list of all the countries with codes, mb I'll need it later. I'll leave a comment.